runtime/doc/lsp.txt

```diff
@@ -865,6 +875,30 @@ apply_workspace_edit({workspace_edit})
 
 buf_clear_diagnostics({bufnr})          *vim.lsp.util.buf_clear_diagnostics()*
                 TODO: Documentation
+ 
+                                        *vim.lsp.util.buf_diagnostics_count()*
+buf_diagnostics_count({kind})
+                Returns the number of diagnostics of given kind for current buffer.
+                Useful for showing diagnostics counts in statusline. eg:
+
+>
+    function! LspStatus() abort
+        let sl = ''
+        if luaeval('server_ready()')
```

Comment:
this triggers an error, shouldn't it be vim.lsp.buf.server_ready ?

Reply:
fixed, thanks